VstsTfsSample/README.md

```diff
@@ -0,0 +1,46 @@
+# VSTS TFS Client Sample
+
+For windows native applications which want to target both VSTS and TFS we recommend using [Azure Active Directory Authentication Library (ADAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-libraries) to generate interactive sign in prompts for VSTS users and [Client Libraries for VSTS and TFS](https://www.visualstudio.com/en-us/docs/integrate/get-started/client-libraries/dotnet) to leverage Windows credential authentication for TFS users.
```

Comment:
That is a good question. It really depends on Will and Joe's answer about Client libraries. If that is something we will continue to try and push by all means. But if we want users to be more familiar with the Azure auth toolset, we need to be pushing them onto the platform. I will likely swap it with client libraries to avoid the text parse issue, but it is a question we need to be able to answer.